spec/mailers/candidate_mailer_ucas_match_reminder_email_spec.rb

```diff
@@ -21,11 +21,10 @@
 
     it_behaves_like(
       'a mail with subject and content',
-      I18n.t!('candidate_mailer.ucas_match_reminder_email.duplicate_applications.subject'),
+      I18n.t!('candidate_mailer.ucas_match_reminder_email.duplicate_applications.subject', withdraw_by_date: '30 November 2020'),
       'heading' => 'Dear Jane',
       'course name and code' => 'Physics (3PH5)',
       'provider' => 'City University',
-      'initial email date' => '16 November 2020',
       'withdrawn by date' => '30 November 2020',
```

Comment:
inconsistent use of key; in other places it is 'withdraw by date'

Reply:
Good idea to tidy this up. I addressed all the comments – thanks!